tests/components/application-connector/test/compass-runtime-agent/application.go

```diff
@@ -0,0 +1,156 @@
+package compass_runtime_agent
+
+import (
+	"github.com/kyma-project/kyma/components/application-operator/pkg/apis/applicationconnector/v1alpha1"
+	"reflect"
+	"testing"
+)
+
+const actualSecretNamespace string = "kyma-integration"
+
+type SecretComparer interface {
```

Comment:
All datatypes and declarations not refering to Application (SecretComparer, Secret and Do) shuold be moved to specific file here: secret.go